playbooks/roles/caasp4/templates/skuba-terraform.tfvars.j2

```diff
@@ -0,0 +1,60 @@
+# This file is managed by ansible and based on terraform.tfvars.ci.example from github.com/SUSE/skuba
+image_name = "{{ image_name }}"
+internal_net = "{{ internal_net }}"
+internal_subnet = "{{ internal_subnet }}"
+internal_router = "{{ internal_router }}"
+external_net = "{{ external_net }}"
+stack_name = "{{ stack_name }}"
+subnet_cidr = "{{ subnet_cidr }}"
+"dns_nameservers" = [
```

Comment:
This should also be a var. Some people don't want to use google dns resolvers :p

Reply:
Removed the nameservers. I think theses should be added (if needed) to our network heat stack when creating the subnet.